flags.go

```diff
@@ -89,6 +90,8 @@ func defineFlagVar() *FlagVar {
 		"Enabling KCP Watcher two-cluster setup to be tested locally using k3d")
 	flag.StringVar(&flagVar.istioNamespace, "istio-namespace", "istio-system",
 		"CLuster Resource Namespace of Istio")
+	flag.StringVar(&flagVar.listenerGatewayPortName, "listener-gateway-port-name", defaultListenerGatewayPortName,
+		"Port name that is specified in the cluster's gateway used for incoming watcher requests")
 	flag.IntVar(&flagVar.listenerHTTPPortLocalMapping, "listener-http-local-mapping", defaultListenerPort,
```

Comment:
The thing is ports `443` and `80` have special meaning, `443` is default for `https`, if you change the local port to `9443`, then it indicates it's https protocol, so the `listenerHTTPPortLocalMapping` better change to `listenerHTTPSPortLocalMapping`. But the current local k3d setup is HTTP only, right? why we use `9443`?